README.md

```diff
@@ -51,6 +51,42 @@ This document provides the information needed to create a DASH packager that is
 
   See https://github.com/google/edash-packager/blob/master/CONTRIBUTING.md for details.
 
+# Using docker for testing/development #
+
```

Comment:
Nice. It will be even better if you can have a brief explanation of what docker is, in one sentence or two.